src/components/grapher/Grapher.vue

```diff
@@ -163,11 +163,12 @@ export default Vue.extend({
       return retVal;
     },
   },
-  mounted: () => {
```

Comment:
I thought this was the preferred way to do things?  Or is `function` preferred?

Reply:
:+1: My bad, for functions in objects, it should look like this:
```ts
{
  thisIsAFunction() {
    ...
  }
}
```